lib/src/core/configs/chip_config.dart

```diff
@@ -5,30 +5,27 @@ import 'package:json_theme/json_theme.dart';
 class ChipConfig extends ComponentConfig {
   const ChipConfig({
     required this.theme,
-    required this.iconTheme,
-    required this.surfaceTintColor,
```

Comment:
Why were these field removed?

Reply:
they are already present in the theme

Reply:
but they are not present in the ThemeDecoder.decodeChipThemeData() method.